grc/gui/canvas/param.py

```diff
@@ -156,6 +160,21 @@ def format_block_surface_markup(self):
         Returns:
             a pango markup string
         """
+
+        if str(self.value) == str(self._evaluated): # This is not a parameter defined by a variable
+            var_value = Utils.encode(self.pretty_print().replace('\n', ' '))
```

Comment:
`var_value` has some other historical meaning in grc code. May I kindly suggest to use maybe `dispaly_value` instead